plaid/errors.py

```diff
@@ -1,10 +1,11 @@
 class PlaidError(Exception):
     retry = False
 
-    def __init__(self, message=None, code=None):
+    def __init__(self, message=None, code=None, requestId=None):
```

Comment:
Can we please make this one `request_id` rather than `requestId`? (same for the property name we set on `PlaidError)